modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1333,8 +1340,7 @@ EXPLAIN INSERT INTO `travel-sample` (key UUID(), value _airport)
               ]
             },
             {
-              "#operator": "Fetch", // <4>
-              "as": "_airport",
+              "#operator": "Fetch",
```

Comment:
Make sure to add the callout numbers back into the examples when you update them
```suggestion
              "#operator": "Fetch", // <4>
```